spec/fstab_spec.rb

```diff
@@ -63,4 +63,50 @@
       subject.instance.write!
     end
   end
+
+  describe "integration test" do
```

Comment:
This is still just testing `#write` though ... right?

Reply:
Not really, it is also testing the parsing, max column length logic, etc.